### DISCUSSION TITLE
Ability to copy a short version of commit hash

### DISCUSSION THREAD

**Discussion by fairking** [gitextensions/gitextensions#9952] [Q&A] [answered]
### Feature description

I pass commit hashes back and forth in chats and forums with my collegues and I would like to easily copy the short version of the commit as it is easier to everyone to deal with that. 
Could you please add such option as well? Let's say first 8 characters or so?

![image](https://user-images.githubusercontent.com/13495631/162758791-36a619fe-a7e6-4dec-94fb-c9da398406ec.png)

We are using Filter field to search such commits in Git Extensions.

![image](https://user-images.githubusercontent.com/13495631/162759354-49f1868c-8757-49bb-9e94-cf5602bf4b6d.png)

Thanks.

### Environment

- GitExtensions version: 3.5.4.12724
- GIT version:  4.4.23(2)
- OS version: Windows 10 x64
- .NET version: v.6

**Comment by vbjay** [comment]
Short hash can point to multiple commits. GitHub translates full sha to
short on pages and makes sure is long enough to be unique.  Passing short
sha is a bad idea. Short sha over time may become non unique when it was
valid before.

**Comment by fairking** [comment]
I know. It was about to make things simple. If I filter out commits and find two branches, it won't be the end of the world.

**Comment by mstv** [comment, marked as answer]
You can create it on your own as a script. A Hotkey can be assigned using the next settings page.

![grafik](https://user-images.githubusercontent.com/36601201/162815444-6f35e5f3-d2b9-406a-8506-95b0233d0472.png)
